dotnet-engine/UnleashEngine/UnleashEngine.cs

```diff
@@ -0,0 +1,103 @@
+﻿
+using System.Runtime.InteropServices;
+using System.Text.Json;
+
+
+namespace Unleash;
+
+public class Context
+{
+    public string? UserId { get; set; }
+    public string? SessionId { get; set; }
+    public string? RemoteAddress { get; set; }
+    public string? Environment { get; set; }
```

Comment:
Question for discussion: can/should we remove this property? Is yggdrasil a good chance to finally lose this deprecated property that is causing a lot of confusion?

Reply:
Oh you sharp eyed ninja. That's a fantastic idea! Yes I believe this can come out now, it'll need a little fiddling in a separate PR though